dependency-check/action.yml

```diff
@@ -0,0 +1,23 @@
+name: Dependency Check
+description: 'OWASP Dependency Check'
+
+runs:
+  using: "composite"
+  steps:
+    - name: Dependency Check
```

Comment:
What are your thoughts on having this do checkout and gradle setup too? That way it's a single drop in action

Reply:
updated.